import/PopupStackView.qml

```diff
@@ -0,0 +1,195 @@
+ *   \li Create a PopupManager from the Popup and add it to a list of 
+ *   PopupManagers
+ *   \li Assign the list mentioned in the above step to the PopupStackView's
+ *   \c popupManagers property
+ * \endlist
+ * 
+ * The PopupStackView is a QML Item that can be anchored, given to a dialog,
+ * or positioned using XY screen coordinates. Transform, Transition, and
+ * other QML animation types can be used to animate the showing and
+ * dismissing of the view.
+ * 
+ * For more information, please see the Popup and PopupManager
+ * documentation.
+ * 
+ * \note Each time a change is made to the Popup, PopupDefinition,
+ * PopupManager, or any of their properties, the popupManagers must be
+ * re-set to the PopupStackView.
+ */
+Control {
+    id: popupStackView
+
+    /*!
+     * \brief A list of PopupManagers that controls the information being 
+     * displayed in a PopupStackView.
+     * \qmlproperty list<PopupManager> popupManagers
+     */
+    property var popupManagers: null
+
+    /*!
+     * \brief This property holds the current top-most item in the stack.
+     * \qmlproperty PopupView currentItem
+     * \sa Stack
+     */
+    property alias currentItem: stack.currentItem
+
+    /*!
+     * \brief This property holds the number of items currently pushed onto the 
+     * stack.
+     * \qmlproperty int depth
+     * \sa Stack
+     */
+    property alias depth: stack.depth
+
+    /*!
+     * \brief This property holds whether a transition is running.
+     * \qmlproperty bool busy
+     * \sa Stack
+     */
+    property alias busy: stack.busy
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that enters the stack when another item is popped off of it.
+     * \qmlproperty Transition popEnter
+     * \sa Stack
+     */
+    property alias popEnter: stack.popEnter
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that exits the stack when the item is popped off of it.
+     * \qmlproperty Transition popExit
+     * \sa Stack
+     */
+    property alias popExit: stack.popExit
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that enters the stack when the item is pushed onto it.
+     * \qmlproperty Transition pushEnter
+     * \sa Stack
+     */
+    property alias pushEnter: stack.pushEnter
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that exits the stack when another item is pushed onto it.
+     * \qmlproperty Transition pushExit
+     * \sa Stack
+     */
+    property alias pushExit: stack.pushExit
+
+    /*!
+     * \brief Signal emitted when an attachment thumbnail is clicked.
+     * The \a index of the PopupAttachment in the PopupAttachmentListModel
+     * of the currently displayed PopupView.
+     */
+    signal attachmentThumbnailClicked(var index)
+
+    clip: true
+
+    implicitWidth: 300 + padding
```

Comment:
do these hard-coded numbers make sense on all screen sizes?

Reply:
They're a very rough estimate of "what looks good" out of the box so the PopupView doesn't start up without drawing anything. They won't suit every situation, but the developers will be doing their own anchoring and custom sizing. 

It's a finger-in-the-air just so it doesn't draw nothing when added to the screen without the widths and heights being explicitly set.